dss/events/handlers/index.py

```diff
@@ -218,9 +221,21 @@ def notify(subscription_id: str, subscription: dict, bundle_id: str, logger):
             "bundle_version": bundle_version
         }
     }
-    # TODO (mbaumann) Ensure webhooks are only delivered over verified HTTPS (unless maybe when running a test)
     callback_url = subscription['callback_url']
-    response = requests.post(callback_url, json=payload)
+
+    assert urlparse(callback_url).scheme in {"http", "https"}
```

Comment:
If a webhook notification fails, it should be clear from the Lambda log why it failed. The caller already performs that logging, yet using `assert` only throws an `AssertionError` with no message indicating the cause of the error / notification failure. I know there is sometimes information that should not be logged for security reasons, yet it seems in the cases where assert is used here, the general cause could be provided without logging sensitive information. Please throw exceptions with a message that will providing meaningful/helpful logging.

Reply:
Will fix.

Reply:
Added more informative assert messages.